### ISSUE TITLE
Labels in chart legends sometimes overlap

### ISSUE THREAD

**Issue by jorisberkhout** [quintel/etmodel#2187]
As can be seen in the chart below, the labels in some chart legends do sometimes overlap.

<img width="610" alt="screen shot 2016-08-16 at 10 38 24" src="https://cloud.githubusercontent.com/assets/4669028/17693132/a219e2c8-639d-11e6-95d1-b4b3db9bffec.png">

I see two solutions:

1. Insert a line break in the label if it gets too long
2. Make the label shorter

Assigning @grdw to see if the first solution mentioned above is possible at all. If this turns out to be impossible, we will need to shorten the label.

**Comment by grdw** [comment]
I came up with a bit of a radical solution for this problem. The legend is drawn as a vector. Therefor all the labels of the legend items have and will always have a fixed width. This is annoying when it comes to text. Therefor I decided to draw all the legends as simple HTML elements.  This will cause it to go from:

![screen shot 2016-09-07 at 14 30 38](https://cloud.githubusercontent.com/assets/2676542/18312025/d6448208-7507-11e6-9ef6-9397f4bafc5a.png)

to:

![screen shot 2016-09-07 at 14 30 43](https://cloud.githubusercontent.com/assets/2676542/18312028/db6d3b80-7507-11e6-8e21-791822f1254d.png)

Now, it doesn't matter how long the label will be, it will always be spaced out equally. The only obvious extra change you're seeing in the 2nd image is that the spacing between the legend items is a bit higher than than the image above. Unfortunately that's a side-effect of using a certain CSS rule which is a little hard to overcome.
